ddmrp/report/mrp_report_bom_structure.py

```diff
@@ -14,6 +14,15 @@ def _get_pdf_doc(self, bom_id, data, quantity, product_variant_id=None):
         )
         return doc
 
+    def _get_bom(
+        self, bom_id=False, product_id=False, line_qty=False, line_id=False, level=False
+    ):
+        res = super()._get_bom(bom_id, product_id, line_qty, line_id, level)
+        bom = self.env["mrp.bom"].browse(bom_id)
+        product = bom.product_id or bom.product_tmpl_id
+        res["lead_time"] = product.produce_delay
+        return res
```

Comment:
This method does not exist upstream since 16.0. Therefore no need to fwport this commit

Reply:
Removed